include/SSVOpenHexagon/Core/MenuGame.hpp

```diff
@@ -23,24 +22,28 @@
 
 #include <SSVUtils/Core/Common/Frametime.hpp>
 
-#include <SFML/Audio.hpp>
-#include <SFML/Graphics.hpp>
-#include <SFML/System.hpp>
-#include <SFML/Window.hpp>
```

Comment:
Please do not remove used `#include` directives :)

Reply:
Oh sorry I did not realize I did!
I truly have no recollection of doing so, I will of course revert it.

Reply:
I do not have recollection of doing so sorry for it.
I will of course restore them.